core/templates/dev/head/pages/header_css_libs.html

```diff
@@ -1,11 +1,10 @@
 <link rel="stylesheet" type="text/css" media="screen"
       href="https://fonts.googleapis.com/css?family=Capriola|Rubik|Roboto|Material+Icons|Open+Sans:400,600">
-{% if not DEV_MODE %}
-  <link rel="stylesheet" type="text/css" media="screen"
-      href="{{get_static_resource_url('/third_party/generated/css/third_party.min.css')}}">
-{% else %}
-  <link rel="stylesheet" type="text/css" media="screen"
-      href="{{get_static_resource_url('/third_party/generated/css/third_party.css')}}">
-{% endif %}
+<link rel="stylesheet" type="text/css" media="screen"
+      href="{{get_static_resource_url('/third_party/generated/css/third_party.min.css')}}"
+      ng-if="!DEV_MODE">
```

Comment:
The ng-if condition should be checked at the starting as that's the initial case.
Likewise for other 2 places